app/views/support/devices/order_status/confirm.html.erb

```diff
@@ -0,0 +1,36 @@
+<%- pass_through_params = {support_enable_orders_form: {order_state: @form.order_state, cap: @form.cap}} %>
+<%- content_for :before_content, govuk_link_to('Back', support_devices_school_enable_orders_path({school_urn: @school.urn}.merge(pass_through_params)), class: 'govuk-back-link') %>
+<%- title = t('page_titles.support_order_status.confirm_enable_orders') %>
+<%- content_for :title, title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-4">
+    <h1 class="govuk-heading-xl">
+      <span class="govuk-caption-xl"><%= @school.name %></span>
+      <%= title %>
+    </h1>
+
+    <%= render Support::EnableOrdersConfirmSummaryListComponent.new(
+      order_state: @form.order_state,
+      cap: @form.cap,
+      allocation: @allocation,
+      change_path: support_devices_school_enable_orders_path(@school.urn, pass_through_params)) %>
+
+    <h2 class="govuk-heading-m govuk-!-margin-top-8">What happens next</h2>
+    <p class="govuk-body">We will:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>email everyone at <%= @school.name %> to tell them they <%= t(@form.order_state, scope: %i[support order_status confirm order_states]) %></li>
+      <li>tell their <%= @school.responsible_body.humanized_type %> <%= @school.responsible_body.name %> that this school <%= t(@form.order_state, scope: %i[support order_status confirm order_states]) %></li>
```

Comment:
I don't think we can include this yet because these things aren't automated. So could we please:

- remove this copy
- cut some cards into the Ready column, to remind us to add in these notifications

Reply:
I've removed the copy in 96b3d4fd  and added card 691](https://trello.com/c/PPq5XlWb/691-add-automated-notifications-to-the-school-and-its-rb-when-the-cap-orderstate-are-changed-via-support)